analyzer/analyzer.go

```diff
@@ -66,3 +91,14 @@ func (src *CallStack) AddCallStack(stack *CallStack) {
 	}
 	src.CallStack.AddCallStack(stack)
 }
+
+type Issues []*Issue
+
+func (issues Issues) Sort() {
```

Comment:
Instead of creating `Issues` and having `Sort` method, better to create independent `SortIssue` method that does the job. We could reuse this in compare function as well.